bin/run_composable-kernels.sh

```diff
@@ -35,8 +37,13 @@ ShouldUpdateCKRepo='no'
 # CK Benchmarks is priate, maybe do not want to update it.
 ShouldUpdateCKBenchmarks='no'
 
-while getopts "hirub" opt; do
-  case $opt in
+# CK may be run using different test- or benchmark-suites.
+SelectedSuite='no'
+ShouldRunCKBenchmarks='no'
+ShouldRunCKClientExamples='no'
```

Comment:
Should we fold this into just the string variable indicating which suite to run? If we cannot specify multiple ones, I currently do not see the benefit of splitting this into a string selector and Boolean vars.
Am I missing something here?